src/icon-tab-bar.scss

```diff
@@ -1127,4 +1200,48 @@ $fd-icon-tab-bar-semantic-values: (
       @include fd-apply-responsive-paddings();
     }
   }
+
+  &-dnd-preview {
+    background: var(--sapObjectHeader_Background, #fff);
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    justify-content: center;
+    font-size: var(--sapFontSize, 0.875rem);
+    line-height: var(--sapContent_LineHeight, 1.4);
+    opacity: 0.8;
+    z-index: 9999 !important;
+
+    * {
+      background: var(--sapObjectHeader_Background, #fff);
+    }
+  }
+
+  &-dnd-hovered {
+    background: var(--sapButton_Hover_Background, #ebf5fe);
+  }
+
+  &-dnd-separator {
+    position: relative;
+
+    &::before {
+      content: '';
+      position: absolute;
+      top: 0;
+      left: -0.5rem;
```

Comment:
shouldn't this be handled by RTL as well?

Reply:
Yes, separator styles don't depend on rtl. These styles for separator between two tabs. Rtl should be handle inside JS for drag and drop.